corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorCoreServer.scala

```diff
@@ -0,0 +1,149 @@
+package org.clulab.processors.coserver
+
+import com.typesafe.config.{ Config, ConfigValueFactory, ConfigFactory }
+import com.typesafe.scalalogging.LazyLogging
+
+import akka.actor._
+import akka.actor.SupervisorStrategy._
+import akka.routing._
+import akka.event.Logging
+
+import org.clulab.processors._
+import org.clulab.processors.bionlp._
+// import org.clulab.processors.clulab._       // TBD: enable when merged
```

Comment:
These were markers for the next task but I realized later it didn't need doing. Thanks for spotting them.